tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -257,6 +257,7 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   pm.addNestedPass<FuncOp>(disc_ral::createDiscAlgebraSimplifierPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscSplitLargeOpsPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscDotRewriterPass());
+  // pm.addNestedPass<FuncOp>(disc_ral::createDiscRngRewriterPass());
```

Comment:
What's the reason for this change?

Reply:
does this PR have anything to do with Rng operation?

Reply:
I accidentally cherr-pick some not related code from my local PoC branch, and removed them in the latest update.